examples/with-segment-analytics/pages/_app.js

```diff
@@ -1,36 +0,0 @@
-import Page from '../components/Page'
```

Comment:
Next.js applications still exist that use the pages router. It would be a good idea to update the documentation for it rather than delete it outright. At the very least, I'd recommend updating the REAMDE to incuding [Segment's own example](https://github.com/segmentio/analytics-next/tree/master/examples/with-next-js), but there's looks a bit complicated... For example, it's not _strictly_ necessary to use partytown or a context provider like they do.

Reply:
I'd be happy to link to their example, but I do agree it is a bit complicated. Alternatively I would also be happy to add a simpler separate example for using Segment with the pages router also.

Reply:
I only happened upon this as I'm updating stuff for work. I feel lucky to have bumped into it while you're devving 😅 

Curious... how different do you think the pages directory setup would be from what you have here? Is `Analytics.load` safe to call in the clear or is Segment's example usage of ContextProvider a necessity to avoid undefined references of `window`?

Reply:
> At the very least, I'd recommend updating the REAMDE to incuding [Segment's own example](https://github.com/segmentio/analytics-next/tree/master/examples/with-next-js)

Segmenter here -- I'd be interested in having the source of truth be your repository. The  example we have is overly complicated as ir more of a playground rather than a super simple vanilla implementation that reflects best practices.

Reply:
> Segment's example usage of ContextProvider a necessity to avoid undefined references of window?

Segmenter here. No need to use ContextProvider if it's not needed -- we only use it in our example because it's more of a playground, so you can do stuff you'd never do in a normal application, like dynamically update the writeKey. 

I'm excited about this example and intend to link to it in the Segment docs/analytics readme as the source of truth for a clean nextjs 13 implementation.

Reply:
it's totally fine to have it live here and differentiate the example by name:

with-segment
with-segment-pages-router

Reply:
I just added a pages router example